libcloud/security.py

```diff
@@ -61,6 +65,19 @@
     '/etc/ssl/certs/YaST-CA.pem',
 ]
 
+# Insert certifi CA bundle path to the front of Libcloud CA bundle search
+# path if certifi is available
+try:
+    import certifi
+except ImportError:
+    has_certifi = False
+else:
+    has_certifi = True
+
+if has_certifi and USE_CERTIFI:
+    certifi_ca_bundle_path = certifi.where()
+    CA_CERTS_PATH.insert(0, certifi_ca_bundle_path)
```

Comment:
I wasn't sure if that was a typo or not, @tonybaloney. So, I stayed quiet.

Yes, 👍 on this change.